src/include/network/remote/dialogs.rb

```diff
@@ -159,8 +159,6 @@ def RemoteMainDialog
       end
       if ret == :next
         CWMFirewallInterfaces.OpenFirewallStore(firewall_widget, "", event)
-        # OpenFirewallStore just sets variables, but we need real firewall write
```

Comment:
well, maybe keep some comment here, so it is obvious that it is not needed to call.

Reply:
if makes sense to you then it's fine for me, what I see there is that no Write method is called, just stored the SuSEFirewall config and the Remote config and then in the return of the dialog is where the Write is done, if it is not clear just reading the code then of course a comment is needed.